app_dart/lib/src/model/appengine/task.dart

```diff
@@ -132,6 +133,11 @@ class Task extends Model {
   @JsonKey(name: 'Reason')
   String reason;
 
+  /// The build ID of luci build.
```

Comment:
Is there an protobuf definition or documentation that explains this field?

Reply:
https://chromium.googlesource.com/infra/luci/luci-go/+/master/buildbucket/proto/build.proto#146
Added to doc.